compiler/cpp/test/test-files/advanced/advanced.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Marc Auberer 2021. All rights reserved.
+
+//
+// Created by Marc on 02.05.2021.
+//
+
+#include "Token.h"
+
+Token::Token() = default;
+
+Token::Token(int type, unsigned int lineNum, unsigned int colNum) {
+    this->type = type;
+    //? if has not . 42 [142] "Identifier (not)" "Keyword (not)" realIdentifier | == != <=> <>=> ==> false true {
```

Comment:
what about some new lines here between `//?` and `if` (each line starting with `             //`)?

Reply:
The cpp compiler does not support this at the moment. Does the Java compiler?

Reply:
yes, it does

Reply:
Have added to my Kanban to add the support for that feature. As soon as it is ready, I will include it here in the tests.